src/layout/Layout.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import Menu from '../components/Menu';
+import MenuItem from '../components/MenuItem';
+import routes from '../routes';
+import { Route, Switch } from 'react-router-dom';
+
+const Layout = props => {
+  const [active, setActive] = useState({});
```

Comment:
```suggestion
  const [, setActive] = useState({});
```
получается как `this.forceUpdate()` :)